definitions/features/downstream.rb

```diff
@@ -30,6 +30,16 @@ def setup_repositories(version)
     end
   end
 
+  def absent_repos(version)
+    all_repos = execute!(
```

Comment:
I guess we should call `subscription-manager refresh` first, similarly as in https://github.com/theforeman/foreman_maintain/pull/148/files

Reply:
Right. It's better to refresh first before listing repos. Done!